src/plugins/blocksmith/components/facebook-post-block-placeholder/index.jsx

```diff
@@ -1,18 +1,24 @@
 import React from 'react';
 import GenericBlockPlaceholder from '@triniti/cms/plugins/blocksmith/components/generic-block-placeholder';
 
-const config = {
-  iconGroup: {
-    icons: {
-      primary: {
-        imgSrc: 'pencil',
-      },
-      secondary: {
-        imgSrc: 'facebook',
+export default (props) => {
+  const { block } = props;
+  const node = block.getData().get('canvasBlock').get('href');
+  const userName = node ? `Posted by ${node.split('/')[3]}` : '';
```

Comment:
@tbone1020 this is not reliable way to get the username. for example, on local (with Joel's recent updates to facebook posts), this is a valid thing to paste in for a fb block: https://www.facebook.com/photo.php?fbid=593811121489641&set=a.223174631886627

which will result in some wacky stuff in the label here

only include it if you can be sure it is a valid user name. looks like vimeo could have one, depending on what is pasted in (because the field is not required you would want to check that it is actually set on the block). couldnt find any others that are promising

Reply:
Changed Facebook placeholder to only show username if present. Also did the same for Vimeo.